pixel/style.css

```diff
@@ -20,5 +20,16 @@ Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, feature
  * Need this until box shadow support added to Gutenberg here: https://github.com/WordPress/gutenberg/pull/43184 
  */
 .pixel-shadow {
-    box-shadow: 5px 5px 0px -2px var(--wp--preset--color--background), 5px 5px var(--wp--preset--color--foreground);
+    box-shadow: var(--wp--custom--shadow)
+}
+
+
+/*
+ * Margin resets for full-width blocks
+ */ 
+.wp-site-blocks .alignfull {
```

Comment:
Should we try to use the `useRootPaddingAwareAlignments` setting? In theory with that setting enabled, we shouldn't need this CSS.

Reply:
Thanks for the suggestion. enabled the above setting that eliminates the custom css. 👍